.github/workflows/pullrequest.yml

```diff
@@ -55,6 +61,17 @@ jobs:
       - name: Run isort formatter check
         run: isort .
 
+      - name: Generate flagd protobuf files
+        run: |
+          cd schemas
+          buf generate buf.build/open-feature/flagd --template protobuf/buf.gen.python.yaml
+          cd ../
+          rm -rf openfeature/contrib/providers/flagd/proto
+          mv proto/python openfeature/contrib/providers/flagd/proto
+          sed -i.bak 's/^from schema.v1 import/from . import/' openfeature/contrib/providers/flagd/proto/schema/v1/*.py
+          rm openfeature/contrib/providers/flagd/proto/schema/v1/*.bak
+          rmdir proto
+
```

Comment:
Could you just run `make grpc`?

Reply:
Yeah, I am going to update the pipeline soon though, so I wouldn't worry too much about it.